<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mcrestool</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdi_area">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1532</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_add_res_file"/>
    <addaction name="action_add_res_folder"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_insert_language"/>
    <addaction name="action_insert_translation_key"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_resource_tree"/>
    <addaction name="action_game_objects"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about_mcrestool"/>
    <addaction name="action_about_qt"/>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_minimize"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_view"/>
   <addaction name="menu_window"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QDockWidget" name="res_tree">
   <property name="windowTitle">
    <string>&amp;Resource Tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dock_widget_contents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="res_tree_view">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="game_objects">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Game &amp;Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dock_widget_contents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="game_objects_view"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As…</string>
   </property>
  </action>
  <action name="action_insert_language">
   <property name="text">
    <string>Insert &amp;Language…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="action_insert_translation_key">
   <property name="text">
    <string>Insert Translation &amp;Key…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_resource_tree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Resource Tree</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_about_mcrestool">
   <property name="text">
    <string>About mcrestool</string>
   </property>
  </action>
  <action name="action_game_objects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Game &amp;Objects</string>
   </property>
  </action>
  <action name="action_minimize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_add_res_file">
   <property name="icon">
    <iconset theme="document-import"/>
   </property>
   <property name="text">
    <string>&amp;Add Resource Pack/Mod…</string>
   </property>
  </action>
  <action name="action_add_res_folder">
   <property name="icon">
    <iconset theme="document-import"/>
   </property>
   <property name="text">
    <string>Add Resource &amp;Folder…</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
